committee-steering/governance/sig-governance.md

```diff
@@ -82,9 +82,22 @@ Subproject contributors (as applicable).
 
 ### Chair
 
+From governance.md:
+> SIG Chairs are intended to be organizers and facilitators, responsible for 
+> the operation of the SIG and for communication and coordination with the 
+> other SIGs, the Steering Committee, and the broader community. 
+
 - Number: 2-3
+- Term: 3 years
+  - 90 days before term is up, Chair will send an email to the SIG mailing list,
+    announce at the regularly scheduled meeting, and send a note to the slack
+    channel asking for interested individuals to step forward. This provides
+    time for outreach, training, and onboarding. 
+  - If no one steps up in 90 days, the current Chair can assume another one (1)
```

Comment:
It would be simpler to have all terms be the same length (I suggest 2y), and permit multiple terms (I suggest unlimited)